deployments/consul-envoy/README.md

```diff
@@ -0,0 +1,79 @@
+# Monitor Consul Connect with Envoy Proxy
+
+Consul Connect emits metrics in statsd format to help understand the network. To report the metrics to SignalFx, you need to configure Consul Connect proxy to send the metrics to StatsD sink exposed by the agent.
+
+First, Configure Consul Connect Proxy to emit StatsD metrics.
+See example Consul service definition below.
+
+```hcl
+services {
+  name = "my_service"
+  port = 8080
+  connect {
+    sidecar_service {
+      proxy {
+        config {
+          envoy_extra_stats_sinks_json = <<EOF
+{
+  "name": "envoy.statsd", 
+  "config": {
+    "address": {
+      "socket_address": {
+        "address": "127.0.0.1",
+        "port_value": 8125,
+        "protocol": "UDP"
+      }
+    },
+    "prefix": "consul.my_service_mesh.my_service"
+  }
+}
+EOF
```

Comment:
Formatting seems weird here.  Also why is there an explicit `EOF`?

Reply:
This is here document syntax supported by hcl https://github.com/hashicorp/hcl

Reply:
oh ok, didn't see that it was a heredoc.